### DISCUSSION TITLE
Multiplayer-Style leaderboard on singleplayer

### DISCUSSION THREAD

**Discussion by GuillermoFEng** [ppy/osu#29829] [Ideas]
Add a feature of making all of your past scores/replays play at the same time you play (at least only the stats) making it look like the multiplayer leaderboard

**Comment by Tenexxt** [comment]
You mean, playing against a ghost that's essentialy your replay?
Sound like something that stable players that hate Lazer with all of their hearts (for whatever reason) would Scream at as it is "useless, not only because it's not on stable but overall" and "they are waiting for pp and framework changes" as I saw this was happening with Daily challenge.

I personally would like for this to be in, but yea.

**Comment by ashandelle** [comment]
I think that this would be a great addition to playlists.
